docs/data/joy/getting-started/usage/usage.md

```diff
@@ -8,15 +8,10 @@ The following code snippet demonstrates a simple app that uses the Joy UI [Butto
 
 ```jsx
 import * as React from 'react';
-import { CssVarsProvider } from '@mui/joy/styles';
 import Button from '@mui/joy/Button';
 
 export default function MyApp() {
```

Comment:
@samuelsycamore after we do the release on Monday, we should unfreeze the sandbox below and be able to remove the CssVarsProvider.